dist/ember-charts.js

```diff
@@ -581,7 +581,7 @@ Ember.Charts.TimeSeriesLabeler = Ember.Mixin.create({
     secondIndex = _.findIndex(allTicks, findTick(labelledTicks[1]));
     firstIndex = _.findIndex(allTicks, findTick(labelledTicks[0]));
     return secondIndex - firstIndex - 1;
-  }).property('xDomain', 'selectedInterval'),
+  }).property('xDomain', 'selectedInterval', 'labelledTicks'),
```

Comment:
When you add an array as a dependent key, you need to be careful.

You can choose between `labelledTicks` and `labelledTicks.[]`